examples/resource_lacework_alert_channel_splunk/main.tf

```
@@ -0,0 +1,12 @@
+provider "lacework" {}
+
+resource "lacework_alert_channel_splunk" "example" {
+  name      = "Splunk Channel Alert Example"
+  hec_token = "AA111111-11AA-1AA1-11AA-11111AA1111A"
```

I found this token in the Splunk docs: https://docs.splunk.com/Documentation/Splunk/8.1.1/Data/UsetheHTTPEventCollector#JSON_request_and_response

```suggestion
  hec_token = "BA696D5E-CA2F-4347-97CB-3C89F834816F"
```